### ISSUE TITLE
Move Applications.DHC under Experimental

### ISSUE THREAD

**Issue by AntoineGautier** [lbl-srg/modelica-buildings#2203]
This issue is to prepare the merge of the `DHC` package into the master, under `Experimental`.

- [ ] Create a branch from `issue1437_district_heating_cooling`
- [ ] Apply a conversion script to
  - move `Applications.CoolingPlants`, `Applications.DHC` and `Applications.DomesticHotWater` under `Experimental`,
  - move `Experimental.DistrictHeatingCooling` under `Obsolete`. But the function `getPeakLoad` currently in `DistrictHeatingCooling` must be moved back into `DHC`, for instance within `Experimental.DHC.Loads.BaseClasses`.
- [ ] Apply `modelica-fmt` to `Experimental.DHC`

Once this is done, we still have to wait for `issue1129_energyPlus_zone` to be merged into the master before opening a PR against the master. Until then, the branch from this issue shall stay alive so that the developers can pull and push against it.

Also,  `issue1437_district_heating_cooling` shall stay alive so that the developers can first pull it before transitioning to the new package location.

**Comment by mwetter** [comment]
@AntoineGautier : I think we could also move the existing `Experimental.DistrictHeatingCooling` to `Buildings.Obsolete`. It is not an architecture that we recommend, so we could think about slowly retiring it.

**Comment by AntoineGautier** [comment]
Agreed.
Plus it will avoid file paths exceeding 130 characters.
I will edit the issue description accordingly.
